### PR TITLE
[resource-manager] Add metrics port

### PR DESCRIPTION
**Pull request by JumasJM**
Type: Improvement

### CODE REVIEW

File: charts/resource-manager/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: resource-manager
 description: A Helm chart for Kubernetes deploying of the Frinx Resource-Manager
 icon: https://avatars.githubusercontent.com/u/23452093?s=200&v=4
 type: application
-version: 2.1.0
```
Comment (Jozefiel) [inline charts/resource-manager/Chart.yaml:6]: 2.2.0 version

File: charts/resource-manager/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
```
Comment (Jozefiel) [inline charts/resource-manager/Chart.yaml:17]: use added kind

File: charts/resource-manager/templates/service.yaml [hunk 3]
```diff
@@ -11,5 +11,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+    - port: {{ .Values.env.RM_METRICS_PORT }}
```
Comment (Jozefiel) [inline charts/resource-manager/templates/service.yaml:14]: use .Values.service.monitorPort

File: charts/resource-manager/templates/deployment.yaml [hunk 4]
```diff
@@ -72,6 +72,9 @@ spec:
             - name: http
               containerPort: {{ .Values.env.RM_API_PORT }}
               protocol: TCP
+            - name: metrics
+              containerPort: {{ .Values.env.RM_METRICS_PORT }}
```
Comment (Jozefiel) [inline charts/resource-manager/templates/deployment.yaml:76]: use .Values.service.monitorPort

File: charts/resource-manager/values.yaml [hunk 5]
```diff
@@ -103,6 +103,7 @@ env:
   RM_API_PORT: 8884
   RM_LOG_PATH: /var/log/rm.log
   RM_LOG_LEVEL: warn
+  RM_METRICS_PORT: 9464
```
Comment (Jozefiel) [inline charts/resource-manager/values.yaml:106]: move it to .Values.service.monitorPort

File: charts/resource-manager/Chart.yaml (hunk 1 above)
Comment (JumasJM) [reply charts/resource-manager/Chart.yaml:6]: resolved

File: charts/resource-manager/Chart.yaml (hunk 2 above)
Comment (JumasJM) [reply charts/resource-manager/Chart.yaml]: resolved

File: charts/resource-manager/templates/service.yaml (hunk 3 above)
Comment (JumasJM) [reply charts/resource-manager/templates/service.yaml]: resolved

File: charts/resource-manager/templates/deployment.yaml (hunk 4 above)
Comment (JumasJM) [reply charts/resource-manager/templates/deployment.yaml]: resolved

File: charts/resource-manager/values.yaml (hunk 5 above)
Comment (JumasJM) [reply charts/resource-manager/values.yaml]: resolved

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**